<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>634</width>
     <height>395</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/jellyfish.jpg</pixmap>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButton_todayDiet">
       <property name="text">
        <string>Today's Diet</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_foodInfo">
       <property name="text">
        <string>Food Info</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_menuInfo">
       <property name="text">
        <string>Menu Info</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_galleryView">
       <property name="text">
        <string>gallery view</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
